frontend/src/components/dialog/DialogForm.vue

```diff
@@ -84,50 +84,22 @@
 <script>
 import { ValidationObserver } from 'vee-validate'
 import ServerError from '@/components/form/ServerError.vue'
+import DialogUiBase from '@/components/dialog/DialogUiBase.vue'
 
 export default {
   name: 'DialogForm',
   components: { ValidationObserver, ServerError },
+  extends: DialogUiBase,
   props: {
-    value: { type: Boolean, required: true },
-
     icon: { type: String, default: '', required: false },
     title: { type: String, default: '', required: false },
 
-    submitAction: { type: Function, default: null, required: false },
-    submitIcon: { type: String, default: 'mdi-check', required: false },
-    submitLabel: {
-      type: String,
-      default: function () {
-        return this.$tc('global.button.submit')
-      },
-      required: false,
-    },
-    submitColor: { type: String, default: 'primary', required: false },
-    submitEnabled: { type: Boolean, default: true, required: false },
-
-    cancelAction: { type: Function, default: null, required: false },
-    cancelIcon: { type: String, default: 'mdi-window-close', required: false },
-    cancelLabel: {
-      type: String,
-      default: function () {
-        return this.$tc('global.button.cancel')
-      },
-      required: false,
-    },
-    cancelColor: { type: String, default: 'secondary', required: false },
-    cancelEnabled: { type: Boolean, default: true, required: false },
-    cancelVisible: { type: Boolean, default: true, required: false },
-
-    loading: { type: Boolean, default: false, required: false },
-    error: { type: [Object, String, Error], default: null, required: false },
-
     maxWidth: { type: String, default: '600px', required: false },
   },
-  data() {
-    return {
-      isSaving: false,
-    }
+  computed: {
+    currentlySaving() {
+      return this.isSaving || this.savingOverride
```

Comment:
Why exactly is the `savingOverride` functionality necessary? Haven't really figured it out by looking at the code.

Reply:
This comes from the problem that DialogForm and DialogBottomSheet are interlocked with DetailPane. If you save and then reduce viewport width, then you could submit again, if the isSaving is not synced across components.

Reply:
Crazy that you thought of this. Is this a real use case though? If the user really wanted to submit twice, they could also open two tabs. So we need consistency validation in the backend anyways. Based on your description, I think we could leave this savingOverride mechanism out if it makes the UI code simpler.

Reply:
It just happened to me 🤷.
I was trying out the feature deployment on the train and resized the window and suddenly it was submittable again, which is confusing ux-wise.

Reply:
Yes, sure, during development. But the average user will never run into this I think, and if they do, it's no big problem if they submit a form twice.

Reply:
Maybe in this case it is not tragic. But other cases the resulting ux may be worse.